### PR TITLE
177580503 split primary account method

### PR DESCRIPTION
**Pull request by maciejkocylapc**
#### What's this PR do?
build split_primary_account method for StakeBossLogic

#### How should this be manually tested?
apart from automatic testing there's a file in /dev folder called stake_boss_demo.rb wich can be used for testing. Please use your own testnet stake address for that.

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/177580503

#### Task completed checklist:
- [Y] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [Y] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: test/logic/stake_boss_logic_test.rb [hunk 1]
```diff
@@ -402,4 +401,150 @@ def create_validators
...
+  end
+
+  def split_primary_account_stub(address:)
+    Proc.new do |arg|
+      cli_req_name = arg[:cli_method].split(' ')[0]
+      case cli_req_name
+      when 'stake-account'
+        json_data = \
+        File.read("#{Rails.root}/test/stubs/solana_stake_account_#{address}.json")
+        {cli_response: json_data, cli_error: nil}
```
Comment (radek-polcode) [inline test/logic/stake_boss_logic_test.rb:537]: add spaces inside, same below

File: app/logic/solana_logic.rb [hunk 2]
```diff
@@ -490,6 +490,8 @@ def cli_request(cli_method, rpc_urls)
         Rails.logger.error "CLI TIMEOUT\n#{e.class}\nRPC URL: #{rpc_url}"
         ''
       end
+      # puts "cli request #{cli_method.split(' ')[0]}\n #{response_json}"
```
Comment (radek-polcode) [inline app/logic/solana_logic.rb:493]: Do we want this comment?

**Review by radek-polcode — CHANGES_REQUESTED** [review]

File: app/logic/solana_logic.rb (hunk 2 above)
Comment (maciejkocylapc) [reply app/logic/solana_logic.rb:493]: I would leave it for now, 'cause it's useful for debugging

**Review by radek-polcode — CHANGES_REQUESTED** [review]
Please apply this https://github.com/brianlong/www.validators.app/pull/91/files#r630187284

**Review by radek-polcode — APPROVED** [review]

**Pull request merged**